orc8r/cloud/deploy/terraform/orc8r-aws/db.tf

```diff
@@ -24,11 +24,12 @@ resource "aws_db_instance" "default" {
 
   vpc_security_group_ids = [aws_security_group.default.id]
 
-  db_subnet_group_name = module.vpc.database_subnet_group
+  db_subnet_group_name = module.vpc.database_subnet_group  
```

Comment:
Throughout this PR you're introducing trailing whitespace

Reply:
:(. Should fix my vscode setting to eliminate this.